### PR TITLE
Fix React warnings in src/app/containers/PageHandlers/withData/index.test.jsx

### LINKED ISSUE #3314: Fix React warnings in src/app/containers/PageHandlers/withData/index.test.jsx
**Describe the bug**
There are React warnings in unit tests.

**To Reproduce**

1. Go to simorgh in terminal
2. Run npm run test:unit
3. Observe React warnings:

```
    console.error src/testHelpers/setupTests.js:53
      Warning: Failed prop type: The prop `data.pageData` is marked as required in `DataContainer`, but its value is `undefined`.
          in DataContainer
    console.error src/testHelpers/setupTests.js:53
      Warning: Failed prop type: The prop `data.dials` is marked as required in `DataContainer`, but its value is `undefined`.
          in DataContainer
```

**Expected behaviour**

No React warning for test suite.

**Testing notes**

No testing required.

### PR DESCRIPTION
**Pull request by FK78**
Resolves #3314 

**Overall change:** _Suppressed prop warnings._

**Code changes:**

- _Added suppressPropWarnings function._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/PageHandlers/withData/index.test.jsx [hunk 1]
```diff
@@ -49,13 +52,15 @@ describe('withData HOC', () => {
   };
 
   describe('with no data', () => {
+    suppressPropWarnings(['data.dials', 'undefined']);
```
Comment (dr3) [inline src/app/containers/PageHandlers/withData/index.test.jsx]: This test isnt testing that dials arnt provided, I think you could just add `dials` to the objects above

File: src/app/containers/PageHandlers/withData/index.test.jsx [hunk 2]
```diff
@@ -49,13 +52,15 @@ describe('withData HOC', () => {
...
     shouldShallowMatchSnapshot(
       'should return the errorMain component and 500 status',
       <WithDataHOC {...noDataProps} />,
     );
   });
 
   describe('with missing articleData', () => {
```
Comment (dr3) [inline src/app/containers/PageHandlers/withData/index.test.jsx]:
```suggestion
  describe('with missing pageData', () => {
```

**Review by dr3 — APPROVED** [review]

**Review by HarveyPeachey — APPROVED** [review]
Nioce

**Pull request merged**